test/relations.jl

```diff
@@ -1614,5 +1617,49 @@ end
     @test all(phase_type .== (nt.phase_type for nt in profiles))
 end
 
+@testset "Thermodynamics - optional args" begin
+    ArrayType = Array{Float64}
+    FT = eltype(ArrayType)
+    param_set = parameter_set(FT)
+    profiles = TestedProfiles.PhaseEquilProfiles(param_set, ArrayType)
+    @unpack ρ, e_int, q_tot, T, phase_type = profiles
+    ts =
+        PhaseEquil_ρeq.(
+            param_set,
+            ρ,
+            e_int,
+            q_tot,
+            8,
+            FT(1e-1),
+            RS.NewtonsMethod,
+            T .+ (FT(0.1) .- rand(FT, length(T))),
+        )
+end
+
+Thermodynamics.error_on_non_convergence() = false
+Thermodynamics.print_warning() = false
+@testset "Thermodynamics - desired settings" begin
+    ArrayType = Array{Float64}
+    FT = eltype(ArrayType)
+    param_set = parameter_set(FT)
+    profiles = TestedProfiles.PhaseEquilProfiles(param_set, ArrayType)
+    @unpack ρ, e_int, q_tot, T, phase_type = profiles
+    ts =
+        PhaseEquil_ρeq.(
+            param_set,
+            ρ,
+            e_int,
+            q_tot,
+            2,
+            FT(1e-1),
+            RS.NewtonsMethod,
+            T .+ (FT(0.5) .- rand(FT, length(T))), # provide an initial guess +/- 0.5 K from true solution
```

Comment:
As above, how about this:
```suggestion
            T .+ (FT(0.5) .* rand(FT, length(T))), # provide an initial guess +/- 0.5 K from true solution
```